src/graphql/graphql-api.php

```diff
@@ -252,6 +247,26 @@ public static function register_types() {
 			]
 		);
 	}
+
+	/**
+	 * Given a block attribute name and value, return a BlockAttribute array.
+	 *
+	 * @param string $name The name of the block attribute.
+	 * @param mixed  $value The value of the block attribute.
+	 *
+	 * @return array
+	 */
+	public static function get_block_attribute_pair( $name, $value ) {
+		// Unknown array types (table cells, for example) are encoded as JSON strings.
+		if ( is_array( $value ) ) {
+			$value = wp_json_encode( $value );
+		}
+
+		return [
```

Comment:
I'm thinking if it would make sense to specify the actual type of `$value` (which would be the best approach imo), or at least specifying that the content is encoded (we may run into performance issues by trying to decode every attribute in frontend)